src/abaqus/UtilityAndView/SymbolicConstant.py

```diff
@@ -42,4 +45,53 @@ def __init__(self, text: str):
         SymbolicConstant
             A :py:class:`~abaqus.UtilityAndView.SymbolicConstant.SymbolicConstant` object.
         """
+        for n in name:
+            if n not in [chr(s) for s in list(range(65,91)) + list(range(48,58))] + ['_']:
+                raise ValueError(f'SymbolicConstant name {name} may only contain upper case, digit or underscore')
```

Comment:
We can actually use the `str.isupper()` to know if all the letters are upper cases.

Reply:
Done in [0a388c4](https://github.com/haiiliin/abqpy/pull/468/commits/0a388c46fb55553d95b451212f8a4a583f2240bb)